src/list-grid.scss

```diff
@@ -332,6 +370,7 @@ $block: #{$fd-namespace}-grid-list;
       width: 0.375rem;
       display: block;
       border-radius: $fd-grid-list-border-radius 0 0 $fd-grid-list-border-radius;
+      z-index: 1;
```

Comment:
why do you need this? We rely on z-index as a last resort

Reply:
In Ngx toolbar component, we have styles present for `fd-toolbar`, which is planned to move to `fundamenatl-styles`.
This class in `toolbar.component.scss` is leaking and causing issue in grid-list component. so `z-index=1` was used to fix.
I have removed it. it will not be needed.